sample/src/main/java/me/saket/cascade/sample/SampleActivity.kt

```diff
@@ -85,6 +85,7 @@ class SampleActivity : AppCompatActivity() {
         }
       }
     }
+    popupMenu.applyBackgroundDim(parent = anchor.rootView, dimAmount = 40)
     popupMenu.show()
```

Comment:
will it be better if dimming can be applied as a parameter to `show()`?

Reply:
Added to show as optional property and default value `null`
Added a null check to ensure it only apply dim if there's something passed to dimAmount.